build/charts/antrea/templates/controller/clusterrole.yaml

```diff
@@ -16,6 +16,7 @@ rules:
       - get
       - watch
       - list
+      - update
```

Comment:
same comment as above. I don't see why the Controller RBAC needs to change

Reply:
I've modified it. "antctl set" command will only be supported in FA pod now.